UIKitTests/UIView/UIViewTests+subviews.swift

```diff
@@ -90,6 +90,16 @@ class UIViewSubviewTests: XCTestCase {
         XCTAssertEqual(view.layer.sublayers!, [subview1.layer, subview2.layer, subview3.layer])
     }
 
+    func testAddSubviewShouldRemoveFromPreviousSuperview() {
+        let prevSuperview = UIView()
+        let subview = UIView()
+        prevSuperview.addSubview(subview)
+        let curtSuperview = UIView()
+        curtSuperview.addSubview(subview)
```

Comment:
nit: could you change `curtSuperview` to `newSuperview`? We don't use that abbreviation elsewhere and it took me some time to realize what it's referring to

Reply:
sure